src/pages/[platform]/build-a-backend/functions/examples/create-user-profile-record/index.mdx

```diff
@@ -37,41 +37,48 @@ You can use `defineAuth` and `defineFunction` to create a [Cognito post confirma
 A user is "confirmed" when they verify their account. Typically this happens when the user confirms their email via the verification email. The post confirmation handler will _not_ be triggered for federated sign-ins (i.e. social sign-in).
 
 </Callout>
-
 To get started, install the `aws-lambda` package, which is used to define the handler type.
 
 ```bash title="Terminal" showLineNumbers={false}
 npm add --save-dev @types/aws-lambda
 ```
 
-Next, create a new directory and a resource file, `amplify/auth/post-confirmation/resource.ts`. Then, define the Function with `defineFunction`:
+Create a new directory and a resource file, `amplify/auth/post-confirmation/resource.ts`. Then, define the Function with `defineFunction`:
 
 ```ts title="amplify/auth/post-confirmation/resource.ts"
 import { defineFunction } from '@aws-amplify/backend';
 
 export const postConfirmation = defineFunction({
-  name: 'postConfirmation',
+  name: 'post-confirmation',
 });
 ```
 
-Then, create the corresponding handler file, `amplify/auth/post-confirmation/post-confirmation/handler.ts`, file with the following contents:
+Use the command below to generate GraphQL client code to call your data backend.
+<Callout info>
+**Note** We are working on bringing the end-to-end typed experience to connect to your data from within function resources without needing this step. If you'd like to provide feedback on the experience or want to have early access, join our [Discord community](https://discord.gg/amplify).
+</Callout>
+
+```sh title="Terminal" showLineNumbers={false}
+npx amplify generate graphql-client-code --out <path-to-post-confirmation-handler-dir>/graphql
```

Comment:
I don't think you can run this command until you run the sandbox command and the data is created in the backend. Then you won't have the file to import in at the next step.